test/runner.server.lua

```diff
@@ -13,7 +13,7 @@ local function findUnitTests(container, foundTests)
 	foundTests = foundTests or {}
 
 	for _, child in ipairs(container:GetChildren()) do
-		if child:IsA("ModuleScript") then
+		if child:IsA("ModuleScript") and not string.match(child.Name, ".*%.spec$") then
```

Comment:
This is a funny trick. I see why, but I thought this condition was backwards the first couple times I looked at it.

Reply:
An approach that we could use instead is changing this function to _not be recursive_, so that anything contained in another folder would not count as a test.

Rust does this by default in order to have a place to put test utilities. It's kind of funny, but it might make sense for this.

Reply:
Yeah, I was debating which of these approaches to take here. I think you're right that that does make more sense, especially since this is the test frameworks tests and shouldn't be quite so numerous that it should be a major problem.